test/pamela/HTN/statements.main.htn.edn

```diff
@@ -31,19 +31,34 @@
   :type :htn-expanded-nonprimitive-task,
   :uid :henpt-16},
  :henpt-17
- {:display-name "Test Of Various Statements",
+ {:args [],
+  :argsmap {},
+  :display-args [],
+  :display-name "Test Of Various Statements",
   :edges [],
   :incidence-set #{},
   :name "test-of-various-statements",
   :type :htn-expanded-nonprimitive-task,
   :uid :henpt-17},
  :hpt-19
  {:args
-  [{:type :field-ref, :names [field1 simple2], :pclass simple-pclass}],
+  [{:type :pclass-ctor,
+    :uid :pclass-2,
+    :pclass simple-pclass,
+    :args ["black"],
+    :fields
+    {simple2 {:access :private, :observable true, :initial "black"}},
+    :ancestry [[:pclass-1 field1]]}],
   :argsmap
   {"x"
-   {:names [field1 simple2], :pclass simple-pclass, :type :field-ref}},
-  :display-args [field1.simple2],
+   {:ancestry [[:pclass-1 field1]],
+    :args ["black"],
+    :fields
+    {simple2 {:access :private, :initial "black", :observable true}},
+    :pclass simple-pclass,
+    :type :pclass-ctor,
+    :uid :pclass-2}},
+  :display-args [(nil "black")],
   :display-name "One Arg Method",
```

Comment:
This raises some interesting issues.

The original call here was `(one-arg-method field1.simple2)`, where the arg was 2 levels of field reference.  
If we had a closed world assumption that allowed us to compile-out the reference value, it would reduce to `(one-arg-method "black")`.  But we can't make this assumption, since the field values (both `field1` and `simple2`) might possibly be altered by the plant.  
I think the best we can do in this case is:
`:display-args [field1.simple2]`